core/src/main/kotlin/org/lflang/generator/ts/TSGenerator.kt

```diff
@@ -94,8 +94,8 @@ class TSGenerator(
 
     init {
         // Set defaults for federate compilation.
-        targetConfig.compiler = "gcc"
-        targetConfig.compilerFlags.add("-O2")
+        targetConfig.compiler.override("gcc")
```

Comment:
Why is the C compiler set in TypeScirpt?

Reply:
876491c71